concourse/Makefile

```
@@ -15,7 +15,7 @@ CERTIFICATION_PIPELINE_NAME ?= pxf-certification
 CLOUDBUILD_PIPELINE_NAME    ?= cloudbuild
 DEV_CLOUDBUILD_PIPELINE_NAME ?= dev-cloudbuild-$(USER)-$(BRANCH)
 PERF_PIPELINE_NAME          ?= pxf_perf-$(SCALE)g-RHEL$(REDHAT_MAJOR_VERSION)
-PIVNET_PIPELINE_NAME        ?= pivnet_artifacts
+GCS_PIPELINE_NAME           ?= gpdb_gcs_artifacts
```

```suggestion
GPDB_ARTIFACTS_PIPELINE_NAME           ?= gpdb-artifacts
```
GCS is too generic of a name for a pipeline that is supposed to fetch GPDB artifacts. Also if we remove GCS from the name, we can change implementation without changing the name / build history in the future

---

> GCS is too generic of a name for a pipeline that is supposed to fetch GPDB artifacts. Also if we remove GCS from the name, we can change implementation without changing the name / build history in the future

Good suggestion. I removed GCS from the name.